LayoutTests/imported/w3c/web-platform-tests/requestidlecallback/callback-timeRemaining-cross-realm-method.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html><!-- webkit-test-runner [ RequestIdleCallbackEnabled=true ] -->
+<!doctype html><!-- webkit-test-runner [ RequestIdleCallbackEnabled=true ] --><!-- webkit-test-runner [ RequestIdleCallbackEnabled=true ] -->
```

Comment:
Is this a typo? This doesn't seem to be upstream (although it's strange that upstream has `webkit-test-runner` in the first place, but separate issue, we should probably just mark RequestIdleCallbackEnabled as testable in another PR)

Reply:
Weird. Upstream has `<!-- webkit-test-runner [ RequestIdleCallbackEnabled=true ] -->` but not sure why it got duplicated here.